hello/App.tsx

```diff
@@ -46,33 +26,47 @@ const Section: React.FC<{
         style={[
           styles.sectionDescription,
           {
-            color: isDarkMode ? Colors.light : Colors.dark,
           },
         ]}>
         {children}
       </Text>
     </View>
   );
 };
+  
+const onLogin = async()=>{
+
+};
 
 const App = () => {
-  const isDarkMode = useColorScheme() === 'dark';
+  const [email, onChangeEmail] = React.useState("");
```

Comment:
quando a gente usa useState a gente costuma usa o prefixo set na função de atualização

```suggestion
  const [email, setEmail] = React.useState("");
```